addons/editor-dark-mode/addon.json

```diff
@@ -116,6 +129,12 @@
       "id": "textShadow",
       "type": "boolean",
       "default": false
+    },
+    {
+      "name": "Toggle dots",
```

Comment:
The setting name should be something like "Show dots" or "Show grid", not "Toggle dots".